src/rdkafka_proto.h

```diff
@@ -523,6 +523,23 @@ typedef struct rd_kafka_buf_s rd_kafka_buf_t;
         RD_UVARINT_ENC_SIZEOF(int32_t)                                 \
         )
 
+#define RD_KAFKAP_MESSAGE_V2_MIN_OVERHEAD                              \
+        (                                                              \
+        /* Length (varint) */                                          \
+        RD_UVARINT_ENC_SIZE_0() +                                      \
```

Comment:
https://travis-ci.org/github/edenhill/librdkafka/jobs/723237176#L246

Suggest changing the ENC_SIZE_0() define to  `((int64_t)1)`

Reply:
Good catch, I used `size_t` instead to silence the signed-unsigned comparison warning.